Userland/Libraries/LibSymbolication/Symbolication.cpp

```diff
@@ -61,24 +63,43 @@ Optional<FlatPtr> kernel_base()
 
 Optional<Symbol> symbolicate(String const& path, FlatPtr address)
 {
-    if (!s_cache.contains(path)) {
-        auto mapped_file = MappedFile::map(path);
-        if (mapped_file.is_error()) {
-            dbgln("Failed to map {}: {}", path, mapped_file.error().string());
+    String full_path = path;
+    if (!path.starts_with('/')) {
+        const char* search_paths[] = { "/usr/lib/{}", "/usr/local/lib/{}" };
+        bool found = false;
+        for (auto& search_path : search_paths) {
+            full_path = String::formatted(search_path, path);
+            int fd = open(full_path.characters(), O_RDONLY, 0);
```

Comment:
Please use `Core::File::exists()` to check if a file exists :)

Reply:
Will do! I'm not familiar yet with the whole library, thanks for pointing this out.